test/CoreTest.cpp

```diff
@@ -841,3 +841,53 @@ TEST_CASE( "no_file_ending", "[core]" )
     REQUIRE_THROWS_WITH(Series("./new_openpmd_output_%05T", Access::CREATE),
                         Catch::Equals("Unknown file format! Did you specify a file ending?"));
 }
+
+TEST_CASE( "custom_geometries", "[core]" )
+{
+    std::vector< int > sampleData( 10, 0 );
+    {
+        Series write( "../samples/custom_geometry.json", Access::CREATE );
+        auto E = write.iterations[ 0 ].meshes[ "E" ];
+        E.setAttribute( "geometry", "customGeometry" );
+        auto E_x = E[ "x" ];
+        E_x.resetDataset( { Datatype::INT, { 10 } } );
+        E_x.storeChunk( sampleData, { 0 }, { 10 } );
+
+        auto B = write.iterations[ 0 ].meshes[ "B" ];
+        B.setGeometry( "customGeometry" );
```

Comment:
Use this in PIConGPU like this:
```suggestion
        B.setGeometry( "other:customGeometry" );
```

Reply:
Should we maybe prepend the prefix unconditionally? So that the call stays `B.setGeometry("customGeometry")`, but `"other:customGeometry"` is written?

Reply:
Yes, that would be even better :)

Reply:
Yeah, I did exactly that now